README.md

```diff
@@ -139,23 +139,28 @@ image.insert_local('localPath' => [ 'Gemfile', 'Rakefile' ], 'outputPath' => '/'
 # => Docker::Image { :id => eb693ec80, :connection => Docker::Connection { :url => tcp://localhost, :options => {:port=>2375} } }
 
 # Add a repo name to Image.
+# docker command for reference: docker tag <IMAGE.ID> base2
 image.tag('repo' => 'base2', 'force' => true)
 # => ["base2"]
 
 # Add a repo name and tag an Image.
+# docker command for reference: docker tag <IMAGE.ID> base2:latest
 image.tag('repo' => 'base2', 'tag' => 'latest', force: true)
 # => ["base2:latest"]
 
 # Get more information about the Image.
+# docker command for reference: docker inspect <IMAGE.ID>
 image.json
 # => {"id"=>"67859327bf22ef8b5b9b4a6781f72b2015acd894fa03ce07e0db7af170ba468c", "comment"=>"Imported from -", "created"=>"2013-06-19T18:42:58.287944526-04:00", "container_config"=>{"Hostname"=>"", "User"=>"", "Memory"=>0, "MemorySwap"=>0, "CpuShares"=>0, "AttachStdin"=>false, "AttachStdout"=>false, "AttachStderr"=>false, "PortSpecs"=>nil, "Tty"=>false, "OpenStdin"=>false, "StdinOnce"=>false, "Env"=>nil, "Cmd"=>nil, "Dns"=>nil, "Image"=>"", "Volumes"=>nil, "VolumesFrom"=>""}, "docker_version"=>"0.4.0", "architecture"=>"x86_64"}
 
 # View the history of the Image.
+# docker command for reference, in fact it mixed the output from two commands: docker image <IMAGE.ID> and docker history <IMAGE.ID>
```

Comment:
The output from `image.history` has no direct docker command to be mapping. The output has the information from  two docker commands: 

```
docker image <IMAGE.ID> 
docker history <IMAGE.ID>
```

Let me think, maybe remove this comment is better.